src/main/kotlin/fi/metatavu/timebank/api/controllers/PersonsController.kt

```diff
@@ -52,26 +52,7 @@ class PersonsController {
 
         keycloakController.updateUsersMinimumBillableRate(keycloakUser, person.minimumBillableRate)
 
-        return Person(
-            id = person.id,
-            firstName = person.firstName,
-            lastName = person.firstName,
-            email = person.email,
-            monday = person.monday,
-            tuesday = person.tuesday,
-            wednesday = person.wednesday,
-            thursday = person.thursday,
-            friday = person.friday,
-            saturday = person.saturday,
-            sunday = person.sunday,
-            active = person.active,
-            unspentVacations = person.unspentVacations,
-            spentVacations = person.spentVacations,
-            minimumBillableRate = keycloakController.getUsersMinimumBillableRate(keycloakUser),
-            language = person.language,
-            startDate = person.startDate,
-            keycloakId = person.keycloakId
-        )
+        return person.copy(minimumBillableRate = keycloakController.getUsersMinimumBillableRate(keycloakUser))
```

Comment:
Now that we have means of finding just one person from Forecast, it would make sense to, instead of returning the same `Person` object we received, to find the person from Forecast again and return that. IMO that more properly mimics a scenario where Persons would be stored in our own database and ensure that the front-end always has up-to-date persons.